### DISCUSSION TITLE
how to use %d ? Such as :  image007.jpg image008..... image015.jpg

### DISCUSSION THREAD

**Discussion by sjh024** [ImageMagick/ImageMagick#7806] [Help] [answered]
### ImageMagick version

Version 7.1.1-41

### Operating system, version and so on

Windows 10 64-bit

### Description

image001.jpg image002.jpg ... image009.jpg can use image00%d.jpg[1-9]
but image007.jpg image008..... image015.jpg how ?

**Comment by dlemstra** [comment]
I have no idea what you are asking us?

**Reply by sjh024** [reply]
Sorry 
Some jpg files  convert  to a PDF.
1. image001.jpg image002.jpg ... image009.jpg can use magick image00%d.jpg[1-9]  a.pdf  . %d can instead One character .
2. image007.jpg image008.jpg...image015.jpg how to express ? ,but 07-15 is two characters,how to instead ?

**Comment by snibgo** [comment, marked as answer]
For two digits with leading zeros, use "%02d", eg "image%02d.jpg". For three digits, use "%03d" etc.

**Comment by sjh024** [comment]
Thx!
